docs/source/introduction.rst

```diff
@@ -127,7 +127,7 @@ dev_mode and PoET.
   * Dev_mode is a simplified random leader algorithm that is useful
     for developers and test networks that require only crash fault tolerance.
 
-  * PoET, short for “Proof of Elapsed Time" is a Nakamoto-style consensus
+  * PoET, short for "Proof of Elapsed Time" is a Nakamoto-style consensus
```

Comment:
This whole introduction needs to be removed and the seth developer's guide moved up a level in the hierarchy.